tools/lint-docstring

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env python3
+
+import argparse
+import glob
+import importlib
+import pathlib
+import sys
+from os.path import basename, dirname, join, relpath
+from typing import Dict, Tuple, cast
+
+
+INITIAL_INDEX = 7
+
+# absolute path to zulip-terminal
+ROOT_DIRECTORY = pathlib.Path(__file__).resolve().parent.parent
+
+# absolute path to zulip-terminal/zulipterminal to be passed as parameter
+ZULIPTERMINAL = join(ROOT_DIRECTORY, "zulipterminal")
+
+# absolute path to the folders under zulipterminal/ to be skipped while parsing
+THEMES_PATH = join(ZULIPTERMINAL, "themes")
+SCRIPTS_PATH = join(ZULIPTERMINAL, "scripts")
+
+# new doc file has been created for the time being to compare it with the original
+DEVELOPER_DOC = join(ROOT_DIRECTORY, "docs", "developer-file-overview.md")
+
+COLUMN_WIDTHS = (23, 20, 103)
+SPACEBAR = " "
+
+# zulipterminal/scripts and zulipterminal/themes do not have any files
+# mentioned in the file-overview. Hence, their description has been
+# added separately.
+SCRIPTS = "zulipterminal/scripts"
+SCRIPTS_DESC = "Scripts bundled with the application"
+THEMES = "zulipterminal/themes"
+THEMES_DESC = "Themes bundled with the application"
+# leaving blank rows after exhausting files from each folder
+BLANK_LINE = f"| {COLUMN_WIDTHS[0]*SPACEBAR}| {COLUMN_WIDTHS[1]*SPACEBAR}| {COLUMN_WIDTHS[2]*SPACEBAR}|\n"
+
+
+def column_text_with_spaces(
+    column_text, columnwise_widths=COLUMN_WIDTHS
+) -> Tuple[str, str, str]:
+    """
+    Returns the text parsed alongwith the adequate amount of spaces
+    after the text
+    """
+    folder_width, file_width, desc_width = columnwise_widths
+    folder_text, file_text, desc_text = column_text
+    folder_remaining_spaces = (folder_width - len(folder_text)) * SPACEBAR
+    file_remaining_spaces = (file_width - len(file_text)) * SPACEBAR
+    desc_remaining_spaces = (desc_width - len(desc_text)) * SPACEBAR
+    folder_name = folder_text + folder_remaining_spaces
+    file_name = file_text + file_remaining_spaces
+    file_desc = desc_text + desc_remaining_spaces
+    return (folder_name, file_name, file_desc)
+
+
+def split_string(long_string):
+    """
+    In case the docstring is longer than the length of description string (>104),
+    this function breaks it into two lines.
```

Comment:
Generally this should represent a 'bug' - we don't want descriptions to be "too long", otherwise the file doesn't represent an overview and the descriptions could get long.

I think you added this to handle the case of the text being multiple lines in one case? If so, as per a comment there, simply make the non-matching text into a comment for now.

Reply:
Yes. It was to handle the case of `color.py` which had a description too long to fit within the docstring column.

Re bug: I'll make changes to throw an error in case the docstring is longer than the column width.